spec/Statements.tex

```diff
@@ -692,8 +692,12 @@ \section{The Use Statement}
   identifier . module-or-enum-name
 
 limitation-clause:
-  `except' identifier-list
-  `only' rename-list
+  `except' exclude-list
+  `only' rename-list[OPT]
+
+exclude-list:
+  identifier-list
+  $ * $
```

Comment:
It puts you into math mode so that * renders as a mid-line star rather than a superscript star.  Build to see.